assets/js/base/components/price-slider/style.scss

```diff
@@ -13,25 +14,25 @@
 	cursor: pointer;
 	z-index: 20;
 	pointer-events: auto;
-	background: $white;
```

Comment:
Was removing this line intentional? It's giving me issues:

<img src="https://user-images.githubusercontent.com/3616980/198675864-e8177ee1-023a-4b9f-a5a6-90ad9042eb11.png" width="337" alt="" />

Reply:
I just moved it on top because the linter complains. Now, I'm noticing that there is `background-color:transparent;`, but this overrides the background rule. I removed it